schemas/groups/electrical.json

```diff
@@ -174,6 +178,16 @@
             }
           }]
         }
+      },
+      "chargeTransferred": {
+        "type": "number",
+        "description": "Amount of charge produced(+) or consumed(-) since last report",
```

Comment:
I still don't get the _since last report_ part.  Couldn't this be something like a trip odometer, reset manually and then keep on accumlating? What does _last report_ actually mean? Last delta? Then it makes a difference if you report this once per second or minute.